setup_rules

```diff
@@ -0,0 +1,24 @@
+#!/bin/env sh
+# Wrapper script for flatpak
```

Comment:
this whole thing is terrifying.

Reply:
I know, I'm very open to better ideas that is somewhat user-friendly in the meantime we wait USB portal

Reply:
implementing said portal?

Reply:
A wiki will be added to guide users to setup rules